entries/QUnit.module.xml

```diff
@@ -15,16 +15,34 @@
 				<desc>Runs after each test</desc>
 			</property>
 		</argument>
+		<argument name="nested" optional="true">
+			<desc>A callback with grouped tests and nested modules to run under the current module label.</desc>
+			<type name="Function">
+				<argument name="hooks" type="Object"></argument>
+			</type>
+		</argument>
 	</signature>
 	<desc>
 		Group related tests under a single label.
 	</desc>
 	<longdesc>
 		<p>
-			All tests that occur after a call to <code>QUnit.module()</code> will be grouped into that module, until other <code>QUnit.module()</code> is called. The test names will all be preceded by the module name in the test results. You can then use that module name to select tests to run.
+			You can then use the module name to organize, select and filter tests to run.
```

Comment:
Why have "then" in this sentence? It's not following anything, so either move the sentence or drop "then." Also, comma after "select."